### PR TITLE
Adding Hosted Che EOL warning

### PR DESCRIPTION
**Pull request by ibuziuk**
Signed-off-by: Ilya Buziuk <ibuziuk@redhat.com>

Try it on Hosted Che
[Contribute](https://che.openshift.io/f?url=https://github.com/ibuziuk/che-website)

image

image

### CODE REVIEW

**Comment by ibuziuk** [comment]
docs related udpate - https://github.com/eclipse/che-docs/pull/1856

**Comment by sunix** [comment]
Could that be more highlighted ? for instance in red ?

File: che/getting-started/cloud-osio/index.php [hunk 1]
```diff
@@ -19,6 +19,7 @@
   <div class="container-fluid">
     <h1>Eclipse Che hosted by Red Hat</h1>
     <h3>Self-service Eclipse Che developer workspaces hosted at che.openshift.io</h3>
+    <blockquote><a href="https://che.openshift.io/">Eclipse Che hosted by Red Hat</a> is going to be shut down on <strong>March 1, 2021 at 00:01 GMT</strong>. A new service based on <a href='https://developers.redhat.com/developer-sandbox#assembly-field-sections-59571'>CodeReady Workspaces</a> is available now. Details about the new service, including signing up for it, can be found <a href='https://developers.redhat.com/developer-sandbox'>here</a>.</blockquote>
```
Comment (benoitf) [inline che/getting-started/cloud-osio/index.php:22]:
```suggestion
    <blockquote><a href="https://che.openshift.io/">Eclipse Che hosted by Red Hat</a> is going to be shut down on <strong>March 1, 2021 at 00:01 GMT</strong>. A new service based on <a href='https://developers.redhat.com/developer-sandbox#assembly-field-sections-59571'>Red Hat CodeReady Workspaces</a> is available now. Details about the new service, including signing up for it, can be found <a href='https://developers.redhat.com/developer-sandbox'>here</a>.</blockquote>
```

File: che/getting-started/cloud/index.php [hunk 2]
```diff
@@ -19,6 +19,7 @@
   <div class="container-fluid">
     <h1>Eclipse Che hosted by Red Hat</h1>
     <h3>Self-service Eclipse Che developer workspaces hosted at che.openshift.io</h3>
+    <blockquote><a href="https://che.openshift.io/">Eclipse Che hosted by Red Hat</a> is going to be shut down on <strong>March 1, 2021 at 00:01 GMT</strong>. A new service based on <a href='https://developers.redhat.com/developer-sandbox#assembly-field-sections-59571'>CodeReady Workspaces</a> is available now. Details about the new service, including signing up for it, can be found <a href='https://developers.redhat.com/developer-sandbox'>here</a>.</blockquote>
```
Comment (benoitf) [inline che/getting-started/cloud/index.php:22]:
```suggestion
    <blockquote><a href="https://che.openshift.io/">Eclipse Che hosted by Red Hat</a> is going to be shut down on <strong>March 1, 2021 at 00:01 GMT</strong>. A new service based on <a href='https://developers.redhat.com/developer-sandbox#assembly-field-sections-59571'>Red Hat CodeReady Workspaces</a> is available now. Details about the new service, including signing up for it, can be found <a href='https://developers.redhat.com/developer-sandbox'>here</a>.</blockquote>
```

File: che/index.php [hunk 3]
```diff
@@ -33,6 +33,7 @@
 <hr class="gray">
 <div class="container-fluid content start-working">
 <h3>Try Eclipse Che using the online service hosted by Red Hat</h3>
+<blockquote><a href="https://che.openshift.io/">Eclipse Che hosted by Red Hat</a> is going to be shut down on <strong>March 1, 2021 at 00:01 GMT</strong>. A new service based on <a href='https://developers.redhat.com/developer-sandbox#assembly-field-sections-59571'>CodeReady Workspaces</a> is available now. Details about the new service, including signing up for it, can be found <a href='https://developers.redhat.com/developer-sandbox'>here</a>.</blockquote>
```
Comment (benoitf) [inline che/index.php:36]:
```suggestion
<blockquote><a href="https://che.openshift.io/">Eclipse Che hosted by Red Hat</a> is going to be shut down on <strong>March 1, 2021 at 00:01 GMT</strong>. A new service based on <a href='https://developers.redhat.com/developer-sandbox#assembly-field-sections-59571'>Red Hat CodeReady Workspaces</a> is available now. Details about the new service, including signing up for it, can be found <a href='https://developers.redhat.com/developer-sandbox'>here</a>.</blockquote>

```

**Review by benoitf — COMMENTED** [review]
as we're using php maybe we could import the content from a single file with include ?

**Comment by ibuziuk** [comment]
@benoitf I'm not very fluent with php includes, but this change will be reverted on March 1st, so I do not think it is a big deal
@sunix have added the red borders:

image

**Review by benoitf — APPROVED** [review]

**Pull request merged**